libnavigation-base/build.gradle

```diff
@@ -25,6 +25,8 @@ dependencies {
     api dependenciesList.mapboxCore
     api dependenciesList.mapboxSdkDirectionsModels
     api dependenciesList.mapboxSdkServicesCore
+    // Navigator
+    api dependenciesList.mapboxNavigator
```

Comment:
```suggestion
    implementation dependenciesList.mapboxNavigator
```

Reply:
Why do mappers live in base now? Couldn't mappers be in core and map from NN types to base SDK types?

Reply:
This isn't a blocker for the PR since mapping happens under the hood, but looks like an incorrect setup. We should follow up if this remains as is.

Reply:
Factory is in base, it uses mappers

Reply:
The factory shouldn't use the mappers through.

The mappers should use the factory to create the SDK objects. This way we can contain all the Nav Native logic in the core module.

Reply:
In any case, could you change this to `implementation` for now?